clients/nodejs/modules/JsonRpcServer.js

```diff
@@ -613,6 +642,27 @@ class JsonRpcServer {
         return block ? this._blockToObj(block, includeTransactions) : null;
     }
 
+    async streamBlocks(ws, query) {
+        const includeTransactions = query && query.has('includeTransactions');
+
+        // Wait until consensus is established
+        if (this._consensusState !== 'established') {
+            await new Promise((resolve) => {
+                this._consensus.on('established', resolve);
+            });
+        }
+
+        // Push every new block to client
+        const listener = this._blockchain.on('head-changed', async () => {
+            const block = await this._getBlockByNumber(this._blockchain.height);
```

Comment:
The `head-changed` event calls its callbacks with the current head blockHead as its first parameter. You could use that.
Otherwise, you can get the head block with  `blockchain.head` directly, instead of via the `getBlockAt` method.

Reply:
Good point!